build.gradle.kts

```diff
@@ -35,6 +35,7 @@ repositories {
             content { groups.forEach { includeGroup(it) } }
         }
     }
+    jcenter()
```

Comment:
This can be an issue, as upgradle is released on Central and should depend only on artifacts that exist on Central. What do you need it for?